scripts/split_eafs.py

```diff
@@ -107,10 +105,8 @@ def read_eaf(ie):
 
         # Check for existence of an annotation in ref tier to silence
         # Annotation value doesn't matter
-        if check_silence_ref_tier:
-            ref_annotation = input_eaf.get_ref_annotation_at_time(silence_tier, start)
-            if len(ref_annotation) is True:
-                skip = True
+        if check_silence_ref_tier and len(input_eaf.get_ref_annotation_at_time(silence_tier, start)):
+            skip = True
```

Comment:
This should work properly now!

Note that you can do (and is preferable):
```python
if skip:
```
There's no need to do `if skip is True`